README.md

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
I still need to review the README and update the instructions so that they're accurate with the new project layout.